Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/AddOwnerFlow.swift

```diff
@@ -106,3 +102,43 @@ class AddOwnerFlowFactory {
         return successVC
     }
 }
+
+class AddOwnerFlowFromSettingsFactory: AddOwnerFlowFactory {
```

Comment:
question: couldn't the super classes' methods used instead of copying and overriding, i.e. to me they look very similar, just thinking that the most of the code repeated here

Reply:
or is it instead of passing different params to the factory in the flow, you're just using different factories and keep the base flow the same?